characterEntities/YellowHero.java

```diff
@@ -16,7 +15,20 @@ public YellowHero(ArrayList<Entity> targets, MapCollisionDetection mapCollisionD
 		setImageIcon("src/assets/hero/yellowNeutral.png");
 		path[0] = CharacterProfile.Path.YELLOW;
 	}
-	
+
+	public boolean evolve (int pathIndex, CharacterProfile.Path path) {
+		if (!super.evolve(pathIndex, path)) return false;
+
+		//temp hack test code, will change in the future
+		if (pathIndex == 1) {
+			setAnimation(pathIndex + 1, new AbilityAnimation(AbilityAnimation.AbilityAnimationType.YELLOW_SECOND, this));
+		} else if (pathIndex == 2) {
+			setAnimation(pathIndex + 1, new AbilityAnimation(AbilityAnimation.AbilityAnimationType.YELLOW_THIRD, this));
+		}
+
+		return true;
+	}
+
 //	public boolean evolve(int path) {
```

Comment:
The evolve functions I added are all temporary and for testing, the old evolve functions still have value, at least until we integrate the stat boosts and class changes